proto/accessgraph/v1alpha/query.proto

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package accessgraph.v1alpha;
+
+import "teleport/legacy/client/proto/event.proto";
+import "teleport/legacy/types/types.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/accessgraph/v1alpha;accessgraphv1alpha";
+
+// AccessGraphService is a service for interacting the access graph service.
+service AccessGraphService {
+  // Query queries the access graph.
+  // Currently only used by WebUI.
+  rpc Query(QueryRequest) returns (QueryResponse);
+  // GetFile gets a static UI file from the access graph container.
+  rpc GetFile(GetFileRequest) returns (GetFileResponse);
+
+  // SendEvent sends an event to the access graph service.
+  rpc SendEvent(SendEventRequest) returns (SendEventResponse);
+
+  // SendResource sends a resource to the access graph service.
+  rpc SendResource(SendResourceRequest) returns (SendResourceResponse);
+}
+
+// Node is a node in the access graph.
+message Node {
+  // ID is the unique ID of the node.
+  string id = 1;
+  // Kind is the kind of the node, ex: "user", "user_group", "resource", etc.
```

Comment:
Is there an exhaustive list of supported resources and their kinds somewhere?

Reply:
We don't have it yet. I'll link it once we have official docs.